test/merchant_test.rb

```diff
@@ -1,34 +1,27 @@
 require 'simplecov'
 SimpleCov.start
-require './lib/merchant'
-require 'rspec'
-
-RSpec.describe Merchant do
+require './lib/helper'
+
+RSpec.describe InvoiceRepository do
```

Comment:
Should be `RSpec.describe Merchant do` right?

Reply:
fixed